src/ntp/Makefile

```diff
@@ -38,8 +38,16 @@ $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 	# and go into learning mode.
 	sed -i 's/\/usr\/sbin\/ntpd {/\/usr\/sbin\/ntpd flags=(attach_disconnected complain) {/' debian/apparmor-profile
 
+ifeq ($(CROSS_BUILD_ENVIRON), y)
+	sed -i 's/dh_auto_configure \--/dh_auto_configure -- --with-yielding-select=yes --with-openssl-incdir=no /g' debian/rules 
```

Comment:
Is there a change to openssl config, --with-openssl-incdir=no?
We have a plan to support FIPS 140-3, it is built on top of the OpenSSl, not sure if it has any impact, could you please explain the behavior change?